### ISSUE TITLE
make check: TypeError: Cannot read property 'onTileErrorStrategy' of undefined

### ISSUE THREAD

**Issue by strk** [CartoDB/Windshaft-cartodb#591]
Upon `make check` from current master branch (b5fccd5bbe3ae3a72c68040e565f17d8f805fa27):
```
Running tests

/usr/src/node/apps/Windshaft-cartodb/Windshaft-cartodb/lib/cartodb/server.js:116
    if (global.environment.enabledFeatures.onTileErrorStrategy !== false) {
                                          ^
TypeError: Cannot read property 'onTileErrorStrategy' of undefined
```

**Comment by strk** [comment]
Running `./configure` before `make check` fixed this (maybe it should be encoded as a dependency)

**Comment by strk** [comment]
Uhm, I'm taking it back, `./configure` does not fix it, still happens (later).
But it *does* fix the same error on `node app.js`, so I'm guessing it does have to do with providing/configuring correct test environment

**Comment by rochoa** [comment]
This seems to be fixed at 0ed46930ddf7d5d01d04210c125a8e487e1dbd39.
